### DISCUSSION TITLE
I found a small bug about Open Systems Pharmacology Suite Manual.

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1389] [Questions & Problems]
Are the legends of these two parts of the modeling concept module labeled backwards?
https://docs.open-systems-pharmacology.org/mechanistic-modeling-of-pharmacokinetics-and-dynamics/modeling-concepts/modeling-concepts-pbpk-modeling-systems-biology
https://docs.open-systems-pharmacology.org/mechanistic-modeling-of-pharmacokinetics-and-dynamics/modeling-concepts/modeling-concepts-pk-and-pd-modeling
![image](https://user-images.githubusercontent.com/86552551/203491098-a92cdd1b-ab21-4d91-851e-69ff3a2fab55.png)
![image](https://user-images.githubusercontent.com/86552551/203491128-8203fee0-a1ff-455e-bcf0-b25d85e66dc8.png)

**Comment by Yuri05** [comment]
Lol, you are right. 
Moving the issue to the documentation repo
